src/Utilities/tests/for_testing/test_checkMatrix.cpp

```diff
@@ -11,58 +11,111 @@
 
 #include "catch.hpp"
 #include "checkMatrix.hpp"
-#include "OhmmsPETE/OhmmsMatrix.h"
+#include "MatrixAccessor.hpp"
+#include <iostream>
+#include <string>
 
+/** \file
+ *  Tests checkMatrix and MatrixAccessor.
+ *  Separate unit tests are likely a waste of effort  since these are complementary
+ *   and in the same module.
+ */
 namespace qmcplusplus
 {
+// When linking containers does cause asan failure this should return
```

Comment:
Is containers still the culprit? Not sure if disabling this test is a good move.

Reply:
I will try this again, but yes containers linked message